app/Http/Controllers/API/RecordsController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Controllers\Controller;
+use Request;
+
+class RecordsController extends Controller
+{
+    public function getRecords(){
+        $records = Record::all();
+
+    return response()->json( $records );
+
+    }
+
+    public function getRecord( $id ){
+        $record = Record::where('id', '=', $id)->first();
+
+    return response()->json( $record );
+    }
+
+    public function postNewRecord( ){
+    $record = new Record();
```

Comment:
This is great, but this will be done through the admin, not here. Remove this function (keep it on the side if you want to work on that admin task and I'll assign it to you 😄)